addons/hr_recruitment_integration_base/views/hr_job_post_views.xml

```diff
@@ -0,0 +1,233 @@
+                        </main>
+                    </t>
+                </templates>
+            </kanban>
+        </field>
+    </record>
+
+    <record id="hr_job_post_kanban_search" model="ir.ui.view">
+        <field name="name">hr.job.post.kanban.search</field>
+        <field name="model">hr.job.post</field>
+        <field name="arch" type="xml">
+            <search>
+                <field name="job_id"/>
+                <field name="platform"/>
+                <field name="apply_vector"/>
+                <separator/>
+                <filter name="filter_published"
+                    string="Published"
+                    domain="[('status', '=', 'success')]"/>
+                <filter name="filter_expired"
+                    string="Expired"
+                    domain="[('status', '=', 'expired')]"/>
+                <filter name="filter_issue"
+                    string="Issue"
+                    domain="[('status', 'in', ['warning', 'failure'])]"/>
+                <separator/>
+                <filter
+                    name="filter_campaign_end_date"
+                    date="campaign_end_date"
+                    string="Expiration date"
+                    default_period="year"
+                    start_year="0"
+                    end_year="2"
+                    start_month="0"
+                    end_month="2"/>
+                <group expand="0" string="Group By">
+                    <filter name="group_platform"
+                        string="Platform"
+                        context="{'group_by': 'platform'}"/>
+                    <filter name="group_job_id"
+                        string="Job"
+                        context="{'group_by': 'job_id'}"/>
+                    <filter name="group_status"
+                        string="Status"
+                        context="{'group_by': 'status'}"/>
+                    <filter name="group_method"
+                        string="Contact Method"
+                        context="{'group_by': 'apply_method'}"/>
+                </group>
+            </search>
+        </field>
+    </record>
+
+    <record id="action_open_hr_job_post" model="ir.actions.act_window">
+        <field name="name">Job Posts</field>
+        <field name="res_model">hr.job.post</field>
+        <field name="view_mode">kanban,tree,form</field>
+        <field name="search_view_id" ref="hr_job_post_kanban_search"/>
+        <field name="view_ids"
+            eval="[(5, 0, 0),
+            (0, 0, {
+                'view_mode': 'kanban',
+                'view_id': ref('hr_recruitment_integration_base.hr_job_post_kanban')
+            }),
+            (0, 0, {
+                'view_mode': 'tree',
+                'view_id': ref('hr_recruitment_integration_base.hr_job_post_tree')
+            }),
+            (0, 0, {
+                'view_mode': 'form',
+                'view_id': ref('hr_recruitment_integration_base.hr_job_post_form')
+            })
+        ]"/>
```

Comment:
Exactly + this could lead to addition issues in the case of view_ids extensions in other applications. This is usually a bad practice, and this should be done instead for each view.
```xml
    <record id="action_open_hr_job_post_view_tree" model="ir.actions.act_window.view">
        <field name="sequence" eval="1"/>
        <field name="view_mode">kanban</field>
        <field name="view_id" ref="hr_job_post_kanban"/>
        <field name="act_window_id" ref="action_open_hr_job_post"/>
    </record>
```